### PR TITLE
feat: Migrate postgresql setup guide

### PR DESCRIPTION
**Pull request by abmusse**
Fixes IBM/ibmi-oss-examples#58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This documentation is open source! Please feel free to help this documentation g
 - [Porting Software to PASE](porting/README.md)
 - [Python Packages](PYTHON_PKGS_GUIDE.md)
 - [ACS Clone Repo Tool](acscloner/README.md)
+- [PostgreSQL](postgresql/installingpostgresqlonibmi.md)
```
Comment (kadler) [inline README.md:25]:
```suggestion
- [PostgreSQL](postgresql/installation.md)
```
The file name seems overly verbose to me.

**Review by kadler — COMMENTED** [review]
Unless we had multiple pages with varying content, we could probably do away with the directory and just have a postgres.md in the top level directory.

File: README.md (hunk 1 above)
Comment (abmusse) [reply README.md:25]: > Unless we had multiple pages with varying content, we could probably do away with the directory and just have a postgres.md in the top level directory.

Lets go with postgres.md in the top level as suggested.

**Review by kadler — APPROVED** [review]
Since my main laptop is out of commission at the moment I can't verify everything looks ok after going through recommonmark, so I'll assume everything is good and we can fix it up later.

**Pull request merged**